pdf-toolbox/src/test/java/com/lowagie/examples/objects/bookmarks/PageLabels.java

```diff
@@ -69,13 +72,11 @@ public static void main(String[] args) {
 
         try (PdfReader reader = new PdfReader("PageLabels.pdf")) {
             final String[] pageLabels = PdfPageLabels.getPageLabels(reader);
-            for (int i = 0; i < pageLabels.length; i++) {
-                System.out.println("pageLabels = " + pageLabels[i]);
-            }
+            assert pageLabels != null;
+            assertThat(Arrays.asList(pageLabels)).containsExactly("i", "ii", "iii", "iv", "1", "2", "3", "A-8", "A-9", "A-10");
             final PdfPageLabels.PdfPageLabelFormat[] pageLabelFormats = PdfPageLabels.getPageLabelFormats(reader);
-            for (int i = 0; i < pageLabelFormats.length; i++) {
-                System.out.println("pageLabelFormats = " + pageLabelFormats[i]);
-            }
+            assert pageLabelFormats != null;
+            assertThat(pageLabelFormats).hasSize(3);
         } catch (DocumentException | IOException de) {
             System.err.println(de.getMessage());
```

Comment:
And also remove exception handling at all, this is not necessary - it's better to get failed test than it will silently log and fail

Reply:
Just do a try-catch in a test, if you want to inspect the exception with some asserts :-)

Reply:
I've removed the try-catch